eZ/Bundle/EzPublishRestBundle/CorsOptions/RestProvider.php

```diff
@@ -42,7 +42,7 @@ public function __construct( RequestMatcherInterface $requestMatcher )
     public function getOptions( Request $request )
     {
         $return = array();
-        if ( $request->attributes->has( 'is_rest_request' ) )
+        if ( $request->attributes->has( 'is_rest_request' ) && true === $request->attributes->get( 'is_rest_request' ) )
```

Comment:
Consistency: I'd rather swap `true` and the `getter`. We never use the <value> === <tested_value> construct.